docs/src/modules/utils/generatePropDescription.ts

```diff
@@ -0,0 +1,151 @@
+import * as doctrine from 'doctrine';
+import * as recast from 'recast';
+import { PropTypeDescriptor } from 'react-docgen';
+import {
+  isElementTypeAcceptingRefProp,
+  isElementAcceptingRefProp,
+} from 'docs/src/modules/utils/generatePropTypeDescription';
+import { DescribeablePropDescriptor } from './createDescribeableProp';
+import { escapeCell } from './helpers';
+
+function resolveType(type: NonNullable<doctrine.Tag['type']>): string {
+  if (type.type === 'AllLiteral') {
+    return 'any';
+  }
+
+  if (type.type === 'VoidLiteral') {
+    return 'void';
+  }
+
+  if (type.type === 'NullLiteral') {
+    return 'null';
+  }
+
+  if (type.type === 'TypeApplication') {
+    return `${resolveType(type.expression)}<${type.applications
+      .map((typeApplication) => {
+        return resolveType(typeApplication);
+      })
+      .join(', ')}>`;
+  }
+
+  if (type.type === 'UnionType') {
+    return type.elements.map((t) => resolveType(t)).join(' | ');
+  }
+
+  if (type.type === 'RecordType') {
+    if (type.fields.length === 0) {
+      return '{}';
+    }
+
+    return `{ ${type.fields.map((field) => resolveType(field)).join(', ')}} `;
```

Comment:
@flaviendelangle I found weird this line. The whitespace at the end and the missing whitespace before the closing curly bracket is a typo?

Reply:
Yes the white space should before the closing bracket